db/migrate/20231115201237_add_role_to_users.rb

```diff
@@ -0,0 +1,5 @@
+class AddRoleToUsers < ActiveRecord::Migration[7.0]
+  def change
+    add_column :users, :role, :string
+  end
+end
```

Comment:
- [ ] Great work adding the role `column` to the `Users` table. Since you are using **_rails enum_** to list out the roles for a `user`, I would advise you to make this column an `integer` or change the **_syntax for listing the enum_** in `app/models/user.rb` file to this:

```ruby
  enum role: {user: 'user', admin: 'admin'}
```

Presently, when I create a user instance, the `role` column is always `nil`. Kindly adopt one of the two solutions I presented above. 💻 😎 

You can read this [article](https://blog.saeloun.com/2022/01/05/how-to-use-enums-in-rails/) to have a better understanding. 📖